src/coinqvest/inc/common/class-common-helpers.php

```diff
@@ -14,7 +14,7 @@ public function get_coinqvest_api_key_secret() {
 		$api_secret = $api_settings['api_secret'];
 
 		if (!isset($api_key) || !isset($api_secret)) {
-			return false;
+			return null;
```

Comment:
It should either return valid credentials or nothing. Not good to do type mixing for return values.